---
title: Wisconsin SARS-CoV-2 (hCoV-19) Genomic Dashboard
subtitle: "enabled by data from <a href='https://www.gisaid.org/'><img src='https://www.gisaid.org/fileadmin/gisaid/img/schild.png' alt='GISAID' style='width:100px'></a>"
layout: dashboard
dashboard-id: 1
img: sarscov2_virus.jpg
thumbnail: sarscov2_virus.jpg
alt: SARS-CoV-2 Virus
anchors: [
Variant Report,
Sequencing Report,
Geographic Report,
Additional Resources
]
---

# Variant Report
<iframe height="725" width="100%" frameborder="no" scrolling="no" src="https://wslhdatacloud.net/variants"><p>Your browser does not support iframes.</p></iframe>

The SARS-CoV-2 virus like all viruses continues to change over time. As the virus replicates and makes copies of itself changes or mutations are introduced. When a virus emerges with one or several new changes it is referred to as a variant of the original virus. Changes in the virus can sometimes lead to a new variant that is better adapted to its environment. The process of the virus changing over time and the selection of new better adapted variants is called virus evolution.

Enabling communication of viral variants requires a process of naming and classifying the variants. Several methods for SARS-CoV-2 variant classification have been developed but the most widely adopted is the [Pangolin](https://cov-lineages.org/resources/pangolin.html) naming scheme, which provides a dynamic naming approach that can adapt to the evolving virus.

# Sequencing Report
<div style="float:left;width:50%">
    <iframe loading="lazy" height="600" width="100%" frameborder="no" scrolling="no" src="https://wslhdatacloud.net/seqTotal"><p>Your browser does not support iframes.</p></iframe>
</div>
<div style="float:right;width:50%">
    <iframe loading="lazy" height="600" width="100%" frameborder="no" scrolling="no" src="https://wslhdatacloud.net/seqTime"><p>Your browser does not support iframes.</p></iframe>
</div>

# Geographic Report 
<iframe loading="lazy" height="650" width="100%" frameborder="no" scrolling="no" src="https://wslhdatacloud.net/varGeo"><p>Your browser does not support iframes.</p></iframe>

# Additional Resources

- [CDC COVID Data Tracker](https://covid.cdc.gov/covid-data-tracker/#variant-proportions)
- [Outbreak.info](https://outbreak.info/)
- [NextStrain.org](https://nextstrain.org/ncov/gisaid/global/6m)
- [CoVariants](https://covariants.org/)
- [cov-lineages.org](https://cov-lineages.org/)

---

GISAID data provided on this website are subject to GISAID’s [Terms and Conditions](https://gisaid.org/terms-of-use/)

We are grateful to the data contributors who shared the data used in this Web Application via the GISAID Initiative: the Authors, the Originating Laboratories responsible for obtaining the specimens, and the Submitting Laboratories that generated the genetic sequences and metadata.

- (a) Elbe, S., and Buckland-Merrett, G. (2017) Data, disease and diplomacy: GISAID’s innovative contribution to global health. Global Challenges, 1:33-46. DOI: [10.1002/gch2.1018](https://doi.org/10.1002/gch2.1018) PMCID: [31565258](https://pubmed.ncbi.nlm.nih.gov/31565258/)

- (b) Shu, Y., McCauley, J. (2017) GISAID: From vision to reality. EuroSurveillance, 22(13) DOI: [10.2807/1560-7917.ES.2017.22.13.30494](https://doi.org/10.2807/1560-7917.es.2017.22.13.30494) PMCID: [PMC5388101](https://pubmed.ncbi.nlm.nih.gov/PMC5388101/)